drivers/ws281x/Kconfig

```diff
@@ -7,16 +7,16 @@
 
 config MODULE_WS281X
     bool "WS2812/SK6812 RGB LED (NeoPixel)"
-    depends on HAS_ARCH_AVR8 || HAS_ARCH_ESP32 || HAS_ARCH_NATIVE
+    depends on CPU_CORE_AVR_MEGA || HAS_ARCH_ESP32 || HAS_ARCH_NATIVE
```

Comment:
Shouldn't this also work on the XMegas, or do the have different cycle times for the same instructions?

Reply:
Currently WS281X implementation is to tight to ATmega. It uses, **atmega_pin_num**, for instance. Besides AVR8 core be the same, peripherals are different between Mega and Xmega. If modules don't use driver low level directly it is possible to keep only **HAS_ARCH_AVR8**. That was initial assumption.